### PR TITLE
Add language fixed

### PR DESCRIPTION
**Pull request by nickbar01234**
# Description

Update language usage in the app. Namely, replace all instances of `president` to `chapter leader` and `students` to `members`.

### CODE REVIEW

File: src/components/user/UserJoinRequest.tsx [hunk 1]
```diff
@@ -72,25 +95,29 @@ const UserJoinRequest = (props: UserJoinRequestProps) => {
                 },
               ]}
               topRightButton={
-                joinRequest == null ? (
+                fetching && fetchingForChapterId === chapter.id ? (
+                  <Spinner width={8} height={8} />
+                ) : joinRequest == null ? (
```
Comment (wkim10) [inline src/components/user/UserJoinRequest.tsx:100]: ===

File: src/components/AdminHomePage.tsx [hunk 2]
```diff
@@ -54,28 +50,18 @@ const AdminHomePage = ({ chapters }: AdminHomePageProps) => {
                 value: chapter.students.length,
               },
               {
-                key: "President",
+                key: "Chapter Leader",
                 value:
                   prez != undefined
```
Comment (wkim10) [inline src/components/AdminHomePage.tsx:55]: not your change but should be !==

File: src/components/user/UserJoinRequest.tsx [hunk 3]
```diff
@@ -56,11 +79,11 @@ const UserJoinRequest = (props: UserJoinRequestProps) => {
               information={[
                 { key: "No. of members", value: chapter.students.length },
                 {
-                  key: "President",
+                  key: "Chapter Leader",
                   value:
                     prez != undefined
```
Comment (wkim10) [inline src/components/user/UserJoinRequest.tsx:84]: again not your change but should be !==

**Review by johnny-t06 — APPROVED** [review]
LGTM, tested Chapter Request and accepting the chapter as an Admin. GJ Nick

**Pull request merged**